sys/net/gnrc/doc.txt

```diff
@@ -0,0 +1,169 @@
+/**
+ * @defgroup    net_gnrc Generic (gnrc) network stack.
+ * @ingroup     net
+ * @brief       RIOT's modular default IP network stack.
+ *
+ * # About
+ *
+ * This module is currently the default network stack of RIOT and includes many components
+ * ranging from a simple mac protocol (@ref net_gnrc_nomac) through a fully-featured IPv6
+ * implementation (@ref net_gnrc_ipv6) with 6LowPAN extensions (@ref net_gnrc_sixlowpan) to
+ * an udp implementation (@ref net_gnrc_udp) and the Routing Protocol for Low Power and Lossy
+ * Networks (@ref net_gnrc_rpl).
+ *
+ * A list of all features contained in the @ref net_gnrc is available in the `Modules` section above.
+ *
+ * # Integration into RIOT-OS
+ *
+ * The integration of the @ref net_gnrc into RIOT-OS is handled by two modules:
```

Comment:
Also this sounds somehow odd. Can you rephrase? Something like `from the application layer GNRC can be accessed through the conn interface, while its interface to the network driver is defined by netdev2` or so.